Makefile

```diff
@@ -1,13 +1,17 @@
-endtoend:
-	npm run cy-e2e
+cy:
```

Comment:
lets remove this file from the PR -- keep it for you on your machine. But, would like to leverage .env files over a makefile